src/sass/utils/_colors.scss

```
@@ -1,5 +1,5 @@
 $color-background: rgba(244, 244, 244, 1);
-$color-background-dark: #2e2e2e;
+$color-background-dark: #686868;
```

це колір бекграунду фігми. треба повернути 
```suggestion
$color-background-dark: #2e2e2e;
```